subsys/net/lib/config/init.c

```diff
@@ -278,6 +278,100 @@ static void setup_ipv6(struct net_if *iface, u32_t flags)
 #define setup_ipv6(...)
 #endif /* CONFIG_NET_IPV6 */
 
+static s8_t global_status;
```

Comment:
What is the purpose of global_status? It is incremented and decremented, but...?